src/vl-map-wms-layer.js

```diff
@@ -0,0 +1,96 @@
+import {VlMapLayer} from '/src/vl-map-layer.js';
+import {
+  OlImageLayer,
+  OlImageWMSSource,
+  OlTileWMSSource,
+  OlTileLayer,
+} from '/node_modules/vl-mapactions/dist/vl-mapactions.js';
+
+/**
+ * VlMapWmsLayer
+ * @class
+ * @classdesc Deze kaartlaag staat toe om een WMS laag aan te maken.
+ *
+ * @extends VlMapLayer
+ *
+ * @property {string} data-vl-name - Attribuut bepaalt de kaartlaag naam.
+ * @property {string} data-vl-url - Attribuut bepaalt de WMS url. Verplicht.
+ * @property {string} data-vl-layers - Attribuut bepaalt de layers van de WMS. Verplicht.
+ * @property {string} [data-vl-styles=] - Attribuut bepaalt de WMS stijlen.
+ * @property {string} [data-vl-version=1.3.0] - Attribuut bepaalt de WMS versie.
+ * @property {number} [data-vl-opacity=1] - Attribuut bepaalt de WMS transparantie.
+ * @property {boolean} [data-vl-tiled=true] - Attribuut bepaalt of de WMS tiled wordt opgehaald.
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-map-wms-layer.html|Demo}
+ */
+export class VlMapWmsLayer extends VlMapLayer {
+  get _url() {
+    return this.getAttribute('data-vl-url') || console.error('url not defined');
```

Comment:
miss hier ook gewoon knallen als de url er niet is. Vermoed dat je zonder url niet veel kaartlaag kan tonen, niet?